modules/installing-ocp-agent-inputs.adoc

```diff
@@ -0,0 +1,162 @@
+// Module included in the following assemblies:
+//
+// * installing-with-agent/installing-with-agent-based-installer.adoc
+
+:_content-type: PROCEDURE
+[id="installing-ocp-agent-inputs_{context}"]
+= Creating the preferred configuration inputs
+
+Use this procedure to create the preferred configuration inputs used to create the agent image.
+
+.Procedure
+
+. Install `nmstate` dependency by running the following command:
++
+[source,terminal]
+----
+$ sudo dnf install /usr/bin/nmstatectl -y
+----
+
+. Place the `openshift-install` binary in a directory that is on your PATH.
```

Comment:
```suggestion
. Place the `openshift-install` binary in a directory that is on your `PATH`.
```
I think PATH should be in backticks.

Reply:
I can't believe I keep missing this one, good catch!  🤦‍♂️